<?xml version="1.0" encoding="UTF-8"?>
<!--
Copyright Â© 2014 Michael Jung

This file is part of Panga.

Panga is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

Panga is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with Panga.  If not, see <http://www.gnu.org/licenses/>.
-->
<ui version="4.0">
 <class>MonteCarloEditor</class>
 <widget class="QWidget" name="MonteCarloEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="left_vertical_layout">
     <item>
      <layout class="QVBoxLayout" name="plot_layout">
       <item>
        <widget class="HistogramPlot" name="plot" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bin_number_layout">
       <item>
        <widget class="QLabel" name="bin_number_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>&amp;Number of Bins:</string>
         </property>
         <property name="buddy">
          <cstring>bin_number_spin_box</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="bin_number_spin_box"/>
       </item>
       <item>
        <widget class="LogarithmicSlider" name="bin_number_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="override_check_box">
         <property name="text">
          <string>&amp;override?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="right_vertical_layout">
     <item>
      <widget class="QLabel" name="select_info_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Press and hold shift to make selections</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="information_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="vertical_spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="text">
        <string>&amp;Clear selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="invert_selection_button">
       <property name="text">
        <string>&amp;Invert selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mask_button">
       <property name="text">
        <string>&amp;Mask with selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="invert_mask_button">
       <property name="text">
        <string>In&amp;vert mask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unmask_button">
       <property name="text">
        <string>&amp;Unmask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_plot_button">
       <property name="text">
        <string>Save &amp;Plot...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogarithmicSlider</class>
   <extends>QSlider</extends>
   <header>logarithmicslider.h</header>
  </customwidget>
  <customwidget>
   <class>HistogramPlot</class>
   <extends>QWidget</extends>
   <header>histogramplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
